apiserver/plane/api/serializers/project.py

```
@@ -9,6 +9,7 @@
 )
 
 from .base import BaseSerializer
+from .issue_type import IssueTypeSerializer
```

Ruff (0.8.2)

`.issue_type.IssueTypeSerializer` imported but unused

Remove unused import: `.issue_type.IssueTypeSerializer`

(F401)